core/src/main/java/tc/oc/pgm/filters/dynamic/FilterMatchModule.java

```diff
@@ -66,13 +67,17 @@
 public class FilterMatchModule implements MatchModule, FilterDispatcher, Tickable, Listener {
 
   private final Match match;
+  private final FeatureDefinitionContext context;
   private final List<Class<? extends Event>> listeningFor = new LinkedList<>();
-  private final PriorityQueue<TimeFilter> timeFilterQueue = new PriorityQueue<>();
+
+  private final Map<Class<? extends ReactorFactory.Reactor>, ReactorFactory.Reactor>
+      activeReactors = new HashMap<>();
```

Comment:
```suggestion
  private final Map<ReactorFactory, ReactorFactory.Reactor>activeReactors = new HashMap<>();
```
Each reactor factory has its reactor